mlapi/discriminating_algo.py

```diff
@@ -96,9 +99,23 @@ def execute_discriminating_facets_algorithm(self, unique_documents_by_facet):
         uniformly_distributed_facets = self.get_uniformly_distributed_facets(facets_sample)
         return self.get_facets_with_max_values(uniformly_distributed_facets)
 
-    def sort_discriminating_facets(self, unique_documents_by_facet):
+    def score_discriminating_facets(self, unique_documents_by_facet):
         values_by_facet = self.get_values_by_facet(unique_documents_by_facet)
         documents_count_by_facet_name = self.get_documents_count_by_facet_name(unique_documents_by_facet)
-        return sorted(values_by_facet.items(),
-                      key=lambda values_by_facet : len(values_by_facet[1]) * documents_count_by_facet_name[values_by_facet[0]],
-                      reverse= True)
\ No newline at end of file
+
+        max_value = 0
+        min_value = 100
```

Comment:
il me semble qu'en python on peu facilement faire "max(array)" et "min(array)", ou dans ton cas
max([len(facetValue) * documents_count_by_facet_name[facetName]] for (facetName, facetValue) in values_by_facet.items()). Mais ça reste pas nécéssairement la meilleure solution car il faudrait pouvoir garder ces valeurs quelque part pour par calculer 2 fois